app/navigation/ObjectivesNavigator.scala

```diff
@@ -49,6 +49,21 @@ class ObjectivesNavigator @Inject()(implicit frontendAppConfig: FrontendAppConfi
 
   private val checkRouteMap: Page => UserAnswers => Call = {
 
+    case CharitableObjectivesPage => userAnswers: UserAnswers => userAnswers.get(CharitableObjectivesPage) match {
```

Comment:
CheckMode changes are affecting coverage so please remove check mode changes or add corresponding test.
Note: Check mode changes are out of scope for this ticket

Reply:
Tests added